staging/src/k8s.io/apiserver/pkg/storage/testing/watcher_tests.go

```diff
@@ -94,6 +100,24 @@ func testWatch(ctx context.Context, t *testing.T, store storage.Interface, recur
 				return nil, fields.Set{"spec.nodeName": pod.Spec.NodeName}, nil
 			},
 		},
+	}, {
+		name:       "filtering",
```

Comment:
FTR - changes in this test are effectively covering what was previously tested with `TestFiltering` test.